### PR TITLE
throwing exception when registry is initialised with invalid conf #25

### PR DESCRIPTION
**Pull request by federicomaffei**
issue #25

### CODE REVIEW

File: .jshintrc [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "newcap"        : false,    // true: Require capitalization of all constructor functions e.g. `new F()`
     "noarg"         : true,     // true: Prohibit use of `arguments.caller` and `arguments.callee`
     "noempty"       : true,     // true: Prohibit use of empty blocks
-    "nonew"         : true,     // true: Prohibit use of constructors for side-effects (without assignment)
+    "nonew"         : false,     // true: Prohibit use of constructors for side-effects (without assignment)
```
Comment (matteofigus) [inline .jshintrc]: uhm, what is this for?

**Comment by matteofigus** [comment]
Can you remove 
```js

if(registry.err){
  return console.log(registry.err);
}
``` 
from the README too? Thanks!

File: .jshintrc (hunk 1 above)
Comment (federicomaffei) [inline .jshintrc]: https://jslinterrors.com/do-not-use-new-for-side-effects
lint was not allowing me to write new oc.Registry(args); without assigning to a variable.

Comment (matteofigus) [inline .jshintrc]: can't you assign a variable there?

**Comment by federicomaffei** [comment]
README done, looking if I can find a way not to change .jshintrc

File: .jshintrc (hunk 1 above)
Comment (federicomaffei) [inline .jshintrc]: done.

**Comment by matteofigus** [comment]
yeah, looks good, thanks

**Pull request merged**